src/NodeServices/TraceSupport.cs

```diff
@@ -17,25 +18,72 @@ public class RegisterForTraceAttribute : Attribute
     /// </summary>
     public static class TraceUtils
     {
+        // ReSharper disable InconsistentNaming
+        //Luke: This is deliberately inconsistent, it is not supposed to be in widespread use, to work around a defiency
+        //in the TLS implementation.
+        //TODO(Luke): Replace this with an attribute lookup
+        internal const string __TEMP_REVIT_TRACE_ID = "{0459D869-0C72-447F-96D8-08A7FB92214B}-REVIT";
```

Comment:
@saintentropy Is this TODO still valid?

Reply:
Not really.  Seems like that ship sailed many years ago